build.gradle

```diff
@@ -136,6 +136,12 @@ dependencies {
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-knn', version: "${opensearch_build}"
     api fileTree(dir: knnJarDirectory, include: '*.jar')
     api group: 'org.opensearch', name:'opensearch-ml-client', version: "${opensearch_build}"
+
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
+    testImplementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
+    testImplementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.1.2'
+    testImplementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.13.3'
+    testImplementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.1.2'
```

Comment:
why do we exactly need these dependencies? As per my understanding all the Jackson related features are abstracted under XContentBuilder in openSearch. You might want to check the usage of these.

Reply:
Checked on XContentBuilder but failed to find the function I would like to use, what I want is to parse string to map or parse a specific field from a json string, please let me know how I can achieve these with this builder. Thanks.

Reply:
For xContentBuilder to map I usually use some combination of: 

1. https://github.com/opensearch-project/neural-search/blob/main/src/test/java/org/opensearch/neuralsearch/plugin/TestUtils.java#L22
2. https://github.com/opensearch-project/k-NN/blob/main/src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java#L154-L164